types/src/zkid.rs

```diff
@@ -225,13 +229,45 @@ impl Claims {
     }
 }
 
-pub type G1 = Vec<String>;
-pub type G2 = Vec<Vec<String>>;
-#[derive(Clone, Debug, Deserialize, PartialEq, Eq, Hash, Serialize)]
+#[derive(
+    Clone, Debug, Deserialize, PartialEq, Eq, Hash, Serialize, CryptoHasher, BCSCryptoHash,
+)]
 pub struct Groth16Zkp {
```

Comment:
Note to self: We'll need to ensure that arkworks deserialization checks subgroup membership properly.

Reply:
iirc they check by default and has those `*_unchecked` apis.

Reply:
confirmed